### PR TITLE
CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by knopki**
Add contributing guide that compatible with #14

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+4. Submit a pull request with a clear and detailed description of your changes.
+
+## Code Style
+
+We use [The Black Code Style](https://black.readthedocs.io/en/stable/the_black_code_style/index.html).
+`black` and `isort` are used as formatters.
+
+## Code Linters
+
+We use almost default configuration of [flake8](https://flake8.pycqa.org/)
```
Comment (blokhin) [inline CONTRIBUTING.md:21]: @knopki why not exactly default? :thinking:

**Review by blokhin — APPROVED** [review]

**Comment by blokhin** [comment]
@knopki would you recommend to use it for the other Metis repos as well, or is it too demanding?

File: CONTRIBUTING.md (hunk 1 above)
Comment (knopki) [reply CONTRIBUTING.md:21]: We set line length to 88 and something more for compatibility with The Black Code Style. Also, relaxing some overly strict rules.

**Comment by knopki** [comment]
> @knopki would you recommend to use it for the other Metis repos as well, or is it too demanding?

Yes, the plan was to make a universal workflow for all repositories.

**Pull request merged**